README.md

```diff
@@ -19,3 +19,102 @@ problems.  Some of the things it does are:
   - Ability to request tasks of the bot, such as setting of labels, assignment,
     and milestone, which would otherwise not be possible without commit rights
     to the repo.
+
+## Development
+
+### Prerequisites
+
+* Membership in the miq-test organization
+* ruby 1.9.3 with bundler
+* redis ~> 2.8.10
+* postgresql
+
+### Steps
+
+1. Fork https://github.com/miq-test/sandbox
+
+2. Fork https://github.com/ManageIQ/miq_bot
+
+3. Clone the miq_bot repo and add the upstream remote:
+    ```
+    git clone git@github.com:<your github handle>/miq_bot.git
+    cd miq_bot
+    git remote add upstream git@github.com:ManageIQ/miq_bot.git
+    git fetch —all
+    ```
+
+4. Add the sandbox app to the `repos` directory so it can be monitored:
+   ```
+   git clone -o upstream git@github.com:miq-test/sandbox.git repos/sandbox
+   ```
+
+5. Install any gem dependencies:
+   ```
+   bundle install
+   ```
+
+6. Open up the `config/database.yml` file in your editor. Add the
+   `username` and `password` for your Postgresql database (If you set
+   up the ManageIQ app locally just use the same credentials).
+
+7. Set up the databases:
+   ```
+   bundle exec rake db:create:all
+   bundle exec rake db:migrate
+   RAILS_ENV=test bundle exec rake db:migrate
+   RAILS_ENV=production bundle exec rake db:migrate
+   ```
+
+8. Add the sandbox app to the database:
+   ```
+   bundle exec rails runner 'CommitMonitorRepo.create_from_github!("miq-test", "sandbox", Rails.root.join("repos", "sandbox"))'
+   ```
+
+9. Either start redis as a daemon or have foreman start it for you by
+   adding this to the `Procfile`:
+   ```yaml
+   # Procfile
+   redis: redis-server /path/to/redis.conf  # change this to the redis.conf provided by your package manager.
+   ```
+
+   You may also want to disable the issue_manager (the thing that
+   responds to commands such as @miq_bot add_label wip) unless you're
+   testing the issue_manager:
+   ```yaml
+   -issue_manager:             bundle exec ruby lib/bot/miq_bot.rb
+   +#issue_manager:             bundle exec ruby lib/bot/miq_bot.rb
+   ```
+
+10. Configure the bot settings. First copy the template:
+    ```
+    cp config/settings.yml config/settings.local.yml
+    ```
+
+    Then set to the following values:
+    ```yaml
+    # config/settings.local.yml:
+
+    # github_credentials
+    username: "miq-bot"
+    password: # token goes here
+
+    # gemfile_checker
+    pr_contacts: [“@your_username"]
+    enabled_repos: ["miq-test/sandbox"]
+
+    # issue_manager
+    repo_names: ["sandbox"]
+
+    # travis_event
+    enabled_repos: ["miq-test/sandbox"]
+    ```
```

Comment:
It's ok as is.  Just need to clean it up so it's easier to configure in the future.